langchain/chains/llm.py

```diff
@@ -184,13 +194,23 @@ async def aapply(
         await run_manager.on_chain_end({"outputs": outputs})
         return outputs
 
-    def create_outputs(self, response: LLMResult) -> List[Dict[str, str]]:
+    @property
+    def _run_output_key(self) -> str:
+        return self.output_key
+
+    def create_outputs(self, response: LLMResult) -> List[Dict[str, Any]]:
```

Comment:
Might break something -- if were possible would be good to avoid introducing similar sounding names (result/response/llm_result)

```suggestion
    def create_outputs(self, llm_result: LLMResult) -> List[Dict[str, Any]]:
```